package.json

```diff
@@ -7,9 +7,9 @@
     "pino-pretty": "./bin.js"
   },
   "scripts": {
-    "ci": "standard && tap 'test/**/*.test.js' --coverage-report=lcovonly",
+    "ci": "standard && tap \"test/**/*.test.js\" --coverage-report=lcovonly",
     "lint": "standard | snazzy",
-    "test": "tap --100 'test/**/*.test.js'"
+    "test": "tap --100 \"test/**/*.test.js\""
```

Comment:
These changes should not have been made. The `"` will introduce shell expansion of `**/*.test.js`. The `'` prevents shell expansion and allows the `glob` package to do the expansion.

@Fdawgs please do not blanket change quotes.

Reply:
Ah, sorry @jsumners. Is there a better way of doing this?

The Fastify repos all use a similar method to ensure the tests run on Windows.

Reply:
They should all be using single quotes.

Reply:
After further investigation, I don't think it matters if single or double quotes are used when it's just globbing (phew, no mass opening of PRs on all of Fastify's repos):

```bash
$ cd test

$ echo *
decorate.test.js hook.test.js integration.test.js plugin.test-d.ts

$ echo "*"
*

$ echo '*'
*
```